sdk/php/template/dagger

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+
+/**
+ * This is the entry point for the module, called from the dagger engine.
+ * Editing this file is highly discouraged and may stop your module from functioning entirely.
+ */
+
+use Symfony\Component\Console\Application;
+use Dagger\Command\EntrypointCommand;
+
+if (file_exists(__DIR__.'/../../autoload.php')) {
+    // The usual location, since this file will reside in vendor/bin
+    require __DIR__.'/../../autoload.php';
+} else {
+    // Useful when doing development on this package
+    require __DIR__.'/vendor/autoload.php';
+}
+
+$console = new Application();
+
+$console->add(new EntrypointCommand());
+$console->setDefaultCommand('dagger:entrypoint');
+
+define('DAG', \Dagger\Dagger::connect());
```

Comment:
I'm not sure about this. Doesn't feel right as a constant. How about just `\Dagger\Dagger::dag()`? Unless you make it a function and it becomes `\Dagger\dag()`?

The point is that multiple calls to it should return the same instance (i.e., not establish a new connection).

Reply:
The function itself is not assigned to the constant, its return value is.

So `DAG` will always be the same instance.

We picked a constant because it is the closest match to how it is used in other SDKs which just call `dag` as a global variable.

Reply:
We now have `sdk/php/src/functions.php` which lets us import the client like so:

```php
//...

use function Dagger\dag;

#[DaggerObject]
class Example
{
     #[DaggerFunction('Echo the value to standard output')]
     public function echo(string $value): Container
     {
         return dag()
             ->container()
             ->from('alpine:latest')
             ->withExec(['echo', $value]);
     }

// ...
```